cmd/testlinter/README.md

```diff
@@ -0,0 +1,100 @@
+# testlinter
+
+testlinter applies different linter rules to test files according to their categories, based on file paths and names.
+It is run as part of the Istio pre-submit linter check. Whitelisting allows rule breaking exceptions, and temporarily
+opt-out.
+
+testlinter is based on [Checker](../README.md), and this package provides the [custom rules](rules) implementation.
+
+## End To End Tests
+
+All "_test.go" files in a "e2e" directory hierarchy are considered as end to end tests.
+
+Example:
+
+```bash
+/istio/tests/e2e/tests/simple/simple_test.go
+```
+
+### End-to-end test rules
+
+1. All skipped tests must be associated with an github issue.
+
+1. All tests should be skipped if testing.short() is true.  This makes it easier to filter out long running tests
```

Comment:
This comment does not seem true, our e2e tests don't do this? Maybe its just disabled. Also I realize you are just moving this over so probably best to not make any changes here in this PR if ever

Reply:
Marked TBD